internal/pkg/source/entry_test.go

```diff
@@ -310,3 +335,183 @@ func getFieldKindToValue(cfg *config.Config, entries []string) map[config.FieldK
 
 	return fieldKindToValue
 }
+
+type TimeFormattingTestCase struct {
+	TestName       string
+	JsonString     string
```

Comment:
```suggestion
	JSONString     string
```

https://google.github.io/styleguide/go/decisions.html#initialisms

Reply:
Thank you!